src/services/oidc-helpers.js

```diff
@@ -82,6 +82,10 @@ export const processSilentSignInCallback = () => {
   new oidc.UserManager().signinSilentCallback()
```

Comment:
Was just looking into this myself. I did find you can use the oidc callbacks in create store module function to capture the framed time out window.

Maybe we should make this return the promise instead? I was actually already given the impression it would of been consistent with `vuexOidcProcessSignInCallback ` which does return a promise.

Reply:
To me, it sounds like a workaround :-). Anyway, iframe window is not timed out. 
It redirects as expected to "<silent_url>#error=login_required". But because we have no fallback there it considered as OK.

Probably you're right and a Promise will be good option too.

Reply:
@coolhome 
But we should consider that library is used by many project/developers and changing library API signature will brake their projects. That is why I decided to add a new method. 
Anyway, things like library API modification (not extension) should be decided by a maintainer. I choose a safe way :-)

@perarnborg FYA